Gopkg.toml

```diff
@@ -42,7 +42,7 @@
 
 [[constraint]]
   name = "github.com/kubernetes-csi/csi-test"
-  version = "2.3.0"
```

Comment:
yes, I reverted 2 commits to get to state before csi-2-3-0 change was merged, just to create leak I knew was there.
My original attempt to create artificial leak by commenting out one remove() did not work, I guess because there is Delete() in After handler.
New attempt seems have worked as Ci failed